crowbar_framework/lib/crowbar/backup/restore.rb

```diff
@@ -163,7 +163,7 @@ def restore_chef
 
               record = JSON.load(file.read)
               filename = file.basename.to_s
-              if proposal?(filename) && type == :databags
+              if proposal?(filename) && !repo?(file) && type == :databags
```

Comment:
I feel a better check is to look if the file is in the crowbar databag; if it's in any other databag, then it's not a proposal. So I'd improve `proposal?` accordingly.